drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -169,14 +171,30 @@ int nrf_wifi_if_send(const struct device *dev,
 		goto out;
 	}
 
-	if ((vif_ctx_zep->if_carr_state != NRF_WIFI_FMAC_IF_CARR_STATE_ON) ||
-	    (!vif_ctx_zep->authorized && !is_eapol(pkt))) {
-		goto out;
-	}
+#ifdef CONFIG_NRF700X_RAWDATA_TX
+	memcpy(&magic_num, pkt->frags->data, sizeof(int));
```

Comment:
typecast `raw_tx_data` ?

Reply:
okay - probably type cast int instead of raw_tx_data

Reply:
done